internal/cmd/shell.go

```diff
@@ -95,32 +96,39 @@ func sendResizeEvents(ctx context.Context, termfd uintptr, process wsep.Process)
 	}
 }
 
-func runCommand(ctx context.Context, envName string, command string, args []string) error {
-	var (
-		entClient = requireAuth()
-	)
+func runCommand(ctx context.Context, envName, command string, args []string) error {
+	entClient := requireAuth()
+
 	env, err := findEnv(ctx, entClient, envName, coder.Me)
 	if err != nil {
-		return err
+		return xerrors.Errorf("find environment: %w", err)
 	}
 
-	termfd := os.Stdout.Fd()
+	termFD := os.Stdout.Fd()
 
-	tty := terminal.IsTerminal(int(termfd))
+	// Check if the client is running an interactive session.
```

Comment:
this seems redundant to the code. 

I'm generally hesitant on adding comments that are redundant to the code in cases like this where the comments just describe what the code does, which just can get out of date.

Reply:
I am not sure most people make the relative between IsTerminal and interactive. We are dealing with somewhat advanced low level programming, following the execution flow in plain engligh helps understanding.

Out of date comments is indeed a problem, but it is on us not to let that happen :)

Reply:
The best way to avoid having out of date comments is to avoid writing code that needs comments :) 

Perhaps the `tty` variable would be more clear if it was named `isInteractive` or something similar. It could also be combined onto the next line. 

I think given how internal/low level (I suppose), I think it's fine